packages/http-proxy-agent/README.md

```diff
@@ -24,6 +24,27 @@ http.get('http://nodejs.org/api/', { agent }, (res) => {
 });
 ```
 
+API
+---
+
+### new HttpProxyAgent(proxy: string | URL, options?: HttpProxyAgentOptions)
+
+The `HttpProxyAgent` class implements an `http.Agent` subclass that connects
+to the specified "HTTP(s) proxy server" in order to proxy HTTP and/or WebSocket
+requests. This is achieved by using the [HTTP `CONNECT` method][CONNECT].
```

Comment:
`http-proxy-agent` does not use `CONNECT`.

Reply:
Oops, I copy/pasted this part without thinking, good catch thanks! Fixed in 301862fc6e20c74945bdfabc9ce1c76143588966